release-notes/opendistro-for-elasticsearch.security.release-notes-1.9.0.0.md

```diff
@@ -0,0 +1,30 @@
+## 2020-06-29 Version 1.9.0.0
+
+Supported Elasticsearch version 7.8.0
+
+### Enhancements
+- Added support for Elasticsearch 7.8.0 ([#516](https://github.com/opendistro-for-elasticsearch/security/pull/516))
+- Allow superadmin to update/delete hidden resources ([#513](https://github.com/opendistro-for-elasticsearch/security/pull/513))
+- Added metadata_body to SAML config ([#477](https://github.com/opendistro-for-elasticsearch/security/pull/477))
```

Comment:
We later renamed it to "metadata_content". Should we merge them to one line.
https://github.com/opendistro-for-elasticsearch/security/pull/495

Reply:
Now I have updated to one line under "Enhancements".  The rename PR was initially under "Maintenance"